source/adios2/ADIOSTypes.inl

```diff
@@ -111,6 +111,12 @@ struct TypeInfo<
     using ValueType = T;
 };
 
+template <typename T, typename Enable>
+inline std::ostream &operator<<(std::ostream &os, const T &value)
+{
+    return os << ToString(value);
+}
+
```

Comment:
Forgive me if I'm not seeing it but what the the `Enable` buy us here that you wouldn't get from the implementation resolving the `ToString` signature?

Reply:
I will look at that again, but I think the `enable_if` may really be required there in order to not overload existing standard library `operator<<` overloads.